tests/test_enterprise/api/test_views.py

```diff
@@ -2027,64 +2028,65 @@ def test_enterprise_customer_support_tool(
     @ddt.data(
         # Request missing required permissions query param.
         (True, False, [], {}, False, {'detail': 'User is not allowed to access the view.'},
-         False, False, False, False, False, False, False),
+         False, False, False, False, False, False, False, False),
```

Comment:
[inform] This whole test case and `@ddt.data` could likely use an overhaul, as it's pretty difficult to reason about all these boolean values for each input case. Opting to continue the current convention, for now, though.